### ISSUE TITLE
Tracks in wrong order

### ISSUE THREAD

**Issue by AaronNGray** [OpenShot/openshot-qt#5661]
**Describe the bug:**
Tracks are in wrong numerbical order. Track 1 should be at the top.

**Steps to reproduce the behavior:**
1. load program
. 
**Expected behavior:**
Tracks should be in numerical order with "Track 1" at the top.

**System Details:**
- OpenShot Version [e.g. 2.4.3]: {{ log_context.openshot_qt_version }}
Any version

- Operating System / Distro: [e.g. Windows 10, Linux Mint 17.1]
Any version

**Screenshots:** (Optional)
Current order
<img width="1280" alt="track-order" src="https://github.com/user-attachments/assets/ac103c45-1b31-443c-9e54-a80ef55f1477">

**Comment by Colorjet3** [comment]
The tracks are not in the wrong order.  This is by design to show you the layered approach that OpenShot works in.  The higher numbered tracks overlay what is below them.  So, if you put something on Track 5 that overlays anything below it and so on.

**Comment by AaronNGray** [comment]
Is it really necessary to have 5 tracks open at the start ? I find even if OpenShot is open full screen window it shows track 5 first, and track 1 is off the bottom of the track pane window. THen dragging a video clip from the project files pane will not scroll the track pane window, meaning that it is necessary to scroll the track pane window down to display track 1 on every first usage of OpenShot. Normally you do not see the large track pane window as in the image above as its normally smaller on startup.

**Comment by Colorjet3** [comment]
I don't think it is necessary to have 5 tracks.  There is an enhancement request to add an option in Preferences to make this user controllable (you decide how many tracks, by default, a new project starts with).

Agreed about some needed improvements in this area.
